frontends/android/BitBoxApp/app/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <color name="colorPrimary">#F5F5F5</color>
-    <color name="colorPrimaryDark">#E5E5E5</color>
+    <color name="colorPrimaryDark">#000000</color>
```

Comment:
With this, the status bar is now always black, in both dark mode and light mode. Before, it was the same color as the BitBoxApp background color, which we added in 3b1d9d78ddae1679a3d44a5102b0207ed9522bcd to make it look much better.

ChatGPT thinks you can have the color depend on dark mode like:

```
 <color name="colorPrimary">#F5F5F5</color>

<!-- For light mode -->
<color name="colorPrimaryDark">#E5E5E5</color>

<!-- For dark mode -->
<color name="colorPrimaryDark" qualifier="night">#333333</color>
```

Maybe you can experiment with this. Obviously the status bar would be wrong if the dark-mode setting in the app is not the same as the system-level dark mode setting, but it is probably better than a black status bar or a status bar that is always light.

Reply:
that's why i wrote

  >Obviously the status bar would be wrong if the dark-mode setting in the app is not the same as the system-level dark mode setting, but it is probably better than a black status bar or a status bar that is always light.

Reply:
For Android, the in-app setting to toggle dark mode could maybe even toggle the Android native dark mod setting for the app, then the above trick would always work too.

https://developer.android.com/develop/ui/views/theming/darktheme

https://developer.android.com/reference/android/app/UiModeManager#setApplicationNightMode(int)